packages/video_player/video_player_avfoundation/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.3.2
+
+* Apply transform accordingly for videos.
```

Comment:
What does "accordingly" mean here? This entry doesn't explain to someone what is actually changing here.

Reply:
Is "Apply the correct transform for videos" seem valid?